src/gsy_e/models/strategy/energy_parameters/heatpump/virtual_heatpump_tank.py

```diff
@@ -103,9 +103,7 @@ class VirtualHeatpumpAllTanksState(AllTanksState):
 
     # pylint: disable=super-init-not-called
     def __init__(self, tank_parameters: List[TankParameters]):
-        self._tanks_states = [
-            VirtualHeatpumpTankState(tank, GlobalConfig.slot_length) for tank in tank_parameters
-        ]
+        self.tanks_states = [VirtualHeatpumpTankState(tank) for tank in tank_parameters]
```

Comment:
Same comment as in the other PR, this probably can be avoided by the followup ticket where we will use the `get_results_dict` output to access the data in dict format instead of directly accessing the classes.